custom_components/healthchecksio/__init__.py

```
@@ -4,7 +4,7 @@
 
 import logging
 
-from homeassistant.config_entries import ConfigEntry
+from homeassistant.config_entries import ConfigEntry, ConfigEntryState, OperationNotAllowed
```

Pylint (3.3.7)

[error] Unable to import 'homeassistant.config_entries'

(E0401)